scripts/ci/install.sh

```diff
@@ -19,26 +19,34 @@
 set -e
 set -u
 
-if [ -z "$TRAVIS_OS_NAME" ]; then
-  echo "This script is only meant to run in Travis CI" 1>&2
+if [ -z "$TRAVIS_OS_NAME" ] && [ -z "$APPVEYOR_REPO_BRANCH" ]; then
+  echo "This script is only meant to run in Travis or Appveyor CI" 1>&2
   exit 1
 fi
 
-if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+if [ -n "$TRAVIS_OS_NAME" ] && [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
     -c "make info && make electron-develop"
 else
   ./scripts/build/check-dependency.sh pip
-  ./scripts/build/check-dependency.sh brew
   ./scripts/build/check-dependency.sh make
   ./scripts/build/check-dependency.sh npm
 
+  if [ -n "$APPVEYOR_REPO_BRANCH" ]; then
```

Comment:
Hmmm, rather than checking for Travis-specific or Appveyor-specific variables throughout the script, maybe it'd be better to set e.g. a `$CI_SERVICE` variable at the top of the script, and then we can add checks for `if [ "$CI_SERVICE"  == "APPVEYOR" ]; then` etc.?

Although it seems the test you're doing here (to determine whether to use `choco` or `brew`) should perhaps also/instead be testing the host OS? Perhaps it'd make sense to pass that in as a cmdline variable?
Or again, perhaps we can automatically determine that from the different env-vars set by the different CI services, so we could have e.g. a `$CI_OS_NAME` variable.